RSE-85 Header X-XSS Protect disabled permanently (#1072)
rundeck/docs: docs/administration/configuration/config-file-reference.md

### Security HTTP Headers

:::warning

> The HTTP header 'XSS-Protection' has been deprecated by mordern browsers and it use can introduce additional security issues on the client side of the application. So Rundeck will not support this header no more since version 4.3.0 .

Reviewer: ```suggestion
The HTTP header 'XSS-Protection' has been deprecated by mordern browsers and it use can introduce additional security issues on the client side of the application. So Rundeck will no longer support this header as of version 4.3.0 .
```

> The HTTP header 'XSS-Protection' has been deprecated by mordern browsers and it use can introduce additional security issues on the client side of the application. So Rundeck will not support this header no more since version 4.3.0 .

Reviewer 2: Replace 2nd Sentence:
"Rundeck has deprecated this setting as of version 4.3.0."

Author: done

#########
# enable x-content-type-options: nosniff  (default: true)

> rundeck.security.httpHeaders.provider.xcto.enabled=trues

Reviewer 2: `trues` doesn't look right.
